setup.py

```diff
@@ -44,6 +44,9 @@
                        'Operating System :: MacOS'],
           platforms='any',
           python_requires='>=3.6',
+          install_requires=[
+              "shtab",
```

Comment:
This should be optional, i.e., only installed with `pip install codespell[completion]` for example

Reply:
However, the code has used shtab in the parse process, how to remove it? Use many if-else or try-catch?

Reply:
Yeah, worst case will you need to dummy the .complete function, is that inserted by shtab?

Reply:
Right.